src/test/java/com/epam/tlmd/pages/MainPage.java

```diff
@@ -1,33 +1,30 @@
 package com.epam.tlmd.pages;
 
+
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.WebElement;
 
 import com.epam.tlmd.util.Init;
 
-public class MainPage {
+public class MainPage extends AbstractPage{
 	
-	 private WebDriver driver;
-	// private final String BASE_URL = "http://www.telemundo.com/";
-	 private final String BASE_URL = Init.setEnviroment();
-	 
+	//final static public String MENU_LINK_XPATH_TEMPLATE = "//a[@class='nav--menu-link' and contains(text(), '%s')]";
+
+	final static public String MENU_LINK_XPATH_TEMPLATE = "(//a[@class='nav--menu-link'])[%s]";
```

Comment:
What do you think will happened if I changed order of links in Menu on the site? It is not good idea to use number to identify necessary link. What if I want to open just Shows hub page? What number should I use?